### DISCUSSION TITLE
Pull request control board

### DISCUSSION THREAD

**Discussion by evbc** [evbc/phase1#40] [General]
Aktuelles zu den PRs

**Comment by evbc** [comment]
Was machen wir mit [PR32](https://github.com/evbc/phase1/pull/32) und mit [28]https://github.com/evbc/phase1/pull/28)  ?

**Reply by Th4tM0nk3y** [reply]
https://github.com/evbc/phase1/pull/28 muss von @mahadyalahow bearbeitet werden, bevor Review und dann Merge möglich sind

**Reply by mahadyalahow** [reply]
alles klar

**Reply by Iryn1234** [reply]
PR32 ist schon approved und rebased worden

**Reply by mahadyalahow** [reply]
brauche noch 1 approval für PR28

**Reply by evbc** [reply]
Scheint niemand mehr da zu sein.

**Comment by evbc** [comment]
https://github.com/evbc/phase1/pull/37 hätte nicht gemerged werden dürfen. Der ist am Review Prozess vorbei geschickt worden!

**Reply by Iryn1234** [reply]
Habe gerade eben reverted
